platform-includes/getting-started-install/react-native.mdx

```diff
@@ -12,7 +6,7 @@ npx @sentry/wizard@latest -s -i reactNative
 
 <Note>
 
-The wizard doesn't support React Native versions `0.68` and older. To use the wizard with older versions of React Native, you'll have to `confirm` in the wizard dialogue and manually patch the Xcode project later. Read more on the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
+The wizard supports React Native versions `0.69` and newer. To use the wizard with older versions of React Native, you'll have to `confirm` in the wizard dialogue and manually patch the Xcode project later. Read more on the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
 
 </Note>
```

Comment:
The latest supported version of RN by Meta is 0.70, I think we can remove this banner. 

https://reactnative.dev/versions